style.css

```diff
@@ -395,6 +401,20 @@ footer .contact-info ul {
   width: 100%;
 }
 
+fieldset {
+  border: 0;
+}
+
+input:focus:invalid {
+  box-shadow: 0 0 5px 1px red;
+  outline: none;
+}
+
+input:focus:valid {
+  box-shadow: 0 0 5px 1px green;
+  outline: none;
+}
+
```

Comment:
- [ ] Please make sure to follow the given design for this section. It helps to make the website more professional, consistent, and good-looking.
Also, it's a good practice for you in order to get used to following design template guidelines.

> Keep in mind that you should follow the given design in typography, font size, font weight, colors, images, spacings (margin and padding), and button interactions such as hover, active (when it's pressed), disabled, and enable (default state).

_Kindly take a look at the images down below, then do the corresponding changes._

### **Your design**
![Screenshot1](https://user-images.githubusercontent.com/47625407/121253742-c4b29c00-c8a9-11eb-901f-6e48bec5632c.png)
### **[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27)**
![Screenshot](https://user-images.githubusercontent.com/47625407/121253798-d5631200-c8a9-11eb-9f9b-06b94a29f7ac.png)

Reply:
I wasn't sure if the red outline was ok just for giving feedback on invalid data, will remove it!